pom.xml

```diff
@@ -389,6 +396,7 @@
                   <include>org.bouncycastle*:*</include>
                   <include>org.lz4*:*</include>
                   <include>commons-io:commons-io:jar:*</include>
+                  <include>io.opentelemetry:*</include> <!-- Add this -->
```

Comment:
Including the opentelemetry interface libraries won't be very useful other than not requiring the dependency at runtime.